modules/local/collect_stats.nf

```diff
@@ -90,17 +101,20 @@ process COLLECT_STATS {
         )
     }
 
+    # Add in stats from taxonomy and function
+    ${read_mergetab}
+
     # Write the table in wide format
     t %>%
         mutate(m = parse_factor(m, levels = TYPE_ORDER, ordered = TRUE)) %>%
         arrange(sample, m) %>%
         pivot_wider(names_from = m, values_from = v) %>%
+        left_join(mergetab) %>%
```

Comment:
```suggestion
        left_join(mergetab, by = 'NN') %>%
```
Change `NN` to whatever field it is. Avoids having distracting messages in the log.